File: JavaBasic/104_language_option.java
// 104	104_language_option.java	Display a language name based on a number option using switch.
public class _104_language_option {
    public static void main(String[] args) {
        int option = 3;
        switch (option) {
            case 1: System.out.println("JavaScript"); break;
            case 2: System.out.println("Python"); break;
            case 3: System.out.println("Java"); break;
            case 4: System.out.println("C++"); break;
            case 5: System.out.println("Rust"); break;
            default: System.out.println("Invalid. Choose 1-5.");
        }
    }
}

File: JSBASIC/104_language_option.js
// 104	104_language_option.js	Display a language name based on a number option using switch.

let option = 3;
console.log("Language option:", option);
switch (option) {
    case 1: console.log("Language: JavaScript"); break;
    case 2: console.log("Language: Python"); break;
    case 3: console.log("Language: Java"); break;
    case 4: console.log("Language: C++"); break;
    case 5: console.log("Language: Rust"); break;
    default: console.log("Invalid option. Choose 1 to 5.");
}
